src/contexts/Photos/Photos.context.tsx

```diff
@@ -312,6 +279,16 @@ export const PhotosContextProvider: React.FC = ({ children }) => {
     await asyncStorageService.savePhotosSyncIsEnabled(enable);
 
     setSyncEnabled(enable);
+
+    if (Platform.OS === 'android') {
```

Comment:
Maybe we can use an enum here, so if it changes, there is no need to replace each check, or a global function

Reply:
this is already a TS union type https://reactnative.dev/docs/platform-specific-code#content